simple-wp-helpdesk/includes/class-cron.php

```
@@ -298,3 +298,113 @@ function swh_process_retention_tickets() {
+	$warn_hours   = swh_get_int_option( 'swh_sla_warn_hours', 4 );
+	$breach_hours = swh_get_int_option( 'swh_sla_breach_hours', 8 );
+	if ( $warn_hours <= 0 && $breach_hours <= 0 ) {
+		return;
+	}
+	$lock_key = 'swh_lock_sla';
+	if ( get_transient( $lock_key ) ) {
+		return;
+	}
+	set_transient( $lock_key, 1, 5 * MINUTE_IN_SECONDS );
+
+	$defs          = swh_get_defaults();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$defs'. (undefined)

(UnusedLocalVariable)